cmd/storagenode/Dockerfile

```diff
@@ -10,6 +10,8 @@ COPY resources/certs.pem /etc/ssl/certs/ca-certificates.crt
 COPY release/${TAG}/storagenode_linux_${GOARCH:-amd64} /app/storagenode
 COPY cmd/storagenode/entrypoint /entrypoint
 COPY cmd/storagenode/dashboard.sh /app/dashboard.sh
+RUN addgroup storj && adduser -D -G storj storj && chown -R storj:storj /app
```

Comment:
Does the directory of binaries need to be owned by the storj user? The user doesn't change the binaries at all and should have access already to run them.

Reply:
Good question. I'm not sure. In case this is read only anyway, the `chown` should be skipped.